src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -2940,6 +2940,26 @@ LogicalResult ONNXGRUOp::inferShapes(
 // Split
 //===----------------------------------------------------------------------===//
 
+LogicalResult ONNXSplitOp::verify() {
+  ONNXSplitOpAdaptor operandAdaptor(*this);
+  for (const auto &operand : operandAdaptor.getOperands())
+    if (!hasShapeAndRank(operand))
+      return success(); // Won't be able to do any checking at this stage.
+
+  auto inputType = operandAdaptor.input().getType().cast<ShapedType>();
```

Comment:
nit use of auto

Reply:
Right, in this case the type is clear because the RHS has a cast, so using auto if OK